en/intro-concepts.md

```diff
@@ -304,6 +304,10 @@ Deploy.
 Allocates (reserves) a node to the MAAS user performing the action (and
 currently logged in). Changes a node's status from 'Ready' to 'Allocated'.
 
+With the CLI, it is necessary to perform this action prior to deploying. With
+the web UI it is done automatically for the user. Acquiring in the web UI is
+used for machine reservation.
```

Comment:
Why do you think there *is* a difference? You either acquire from the CLI or you acquire from the web UI. What *I* do not like is how the CLI uses the command `allocate` and the web UI uses the action `acquire`. Both lead to a node being left with a status of `Allocated`. It's another example of bad usage of terminology.